### PR TITLE
New simple footer

### PR DESCRIPTION
**Pull request by mwernimont**
@jiwalker-usgs want to give it a look?

### CODE REVIEW

File: inst/templates/footer.mustache [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
+      <a href="mailto:vizlab@usgs.gov">
+        <p>Contact Us</p>
+      </a>
+    </div>
+    
     <div id="relatedVizzies">
-    	<h3>Related Visualizations:</h3>
-
-    	{{#vizzies}}
+      <!--{{#vizzies}}-->
```
Comment (jiwalker-usgs) [inline inst/templates/footer.mustache]: What is happening here?

File: inst/resource.library.yaml [hunk 2]
```diff
@@ -160,6 +161,11 @@
   mimetype: text/javascript
   publisher: resource
 -
+  id: lib-header-css
```
Comment (jiwalker-usgs) [inline inst/resource.library.yaml]: Is there a header template that uses this?

**Comment by mwernimont** [comment]
@jiwalker-usgs The  was a forgotten attempt at getting the github blocking to stop, but didn't see it since mustache files have no code coloring. Removed the .crunch file too. For the header I don't know, do we have a header template that could use it?

File: inst/templates/footer.mustache [hunk 3]
```diff
@@ -1,29 +1,43 @@
...
-    					<img src="{{thumbLoc}}" alt=""/>
-    				</div>
     			</div>
     		</a>
     	{{/vizzies}}
     </div>
-  
-  {{#blogsInFooter}}
+    
+   <!-- {{#blogsInFooter}}
```
Comment (jiwalker-usgs) [inline inst/templates/footer.mustache]: You also commented out blogsInFooter.

Comment (mwernimont) [inline inst/templates/footer.mustache]: Yep, Emily wasn't sure what we are doing with blogs in the footer, so figured commenting it out was the best solution for the moment

Comment (jiwalker-usgs) [inline inst/templates/footer.mustache]: Not a huge fan of commented out code, but ok for now.

**Pull request merged**